components/runtime/squeak4/history

```diff
@@ -0,0 +1,3 @@
+runtime/squeak-4@4.19.6-2020.0.1.4 runtime/smalltalk/squeak
+runtime/squeak-4-nodisplay@4.19.6-2020.0.1.4 runtime/smalltalk/squeak-nodisplay
```

Comment:
All your FMRIs are missing ",5.11-" inside, eg.
runtime/squeak-4@4.19.6-2020.0.1.4 runtime/smalltalk/squeak should be
runtime/squeak-4@4.19.6,5.11-2020.0.1.4 runtime/smalltalk/squeak

This is the <release> part of an FMRI and usually omitted when being shown.